sdk/servicebus/service-bus/samples/javascript/package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "azure-storage-blob-samples-js",
```

Comment:
Thanks 😂

Reply:
Yeah....I mean, the blob samples are good. But not so good they should go everywhere... ;)